src/controller/genericmidi.cxx

```diff
@@ -211,240 +211,15 @@ void GenericMIDI::trackSendActive(int t, int send, bool a)
 
 void GenericMIDI::trackJackSend(int t, float v)
 {
-
+// TODO
 }
 
 void GenericMIDI::trackJackSendActivate(int t, bool a)
 {
-
-}
-
-/*
-void GenericMIDI::progress(int t, int s, float f)
-{
-  
-  unsigned char data[3];
-  data[0] = 176;
-  data[1] = 48; // record enable LED
-  data[2] = 127 * f;
-  jack->midiObserverWriteMIDI( _port,  &data[0] );
-  
-}
-
-void GenericMIDI::trackSend(int t, int send, float v)
-{
-  if ( t >= NTRACKS)
-  {
-    // master track
-    return;
-  }
-  
-  unsigned char data[3];
-  
-  if ( send == SEND_KEY )
-  {
-    data[0] = 144 + t;
-    data[1] = 49;
-    data[2] = v > 0.5 ? 127 : 0 ;
-  }
-  else if ( send == SEND_XSIDE )
-  {
-    data[0] = 176 + t;
-    data[1] = 16;
-    data[2] = 127 * v;
-  }
-  else if ( send == SEND_POSTFADER )
-  {
-    data[0] = 176 + t;
-    data[1] = 17;
-    data[2] = 127 * v;
-  }
-  
-  jack->midiObserverWriteMIDI( _port,  &data[0] );
-}
-
-void GenericMIDI::launchScene( int s )
-{
-  unsigned char data[3];
-  
-  for(int i = 0; i < 5; i++ )
-  {
-    data[0] = 128;
-    data[1] = 82 + i; // scene play
-    data[2] = 0;
-    jack->midiObserverWriteMIDI( _port,  &data[0] );
-  }
-  
-  data[0] = 144;
-  data[1] = 82 + s;
-  data[2] = 127;
-  jack->midiObserverWriteMIDI( _port,  &data[0] );
-}
-
-void GenericMIDI::mute(int t, bool b)
-{
-  
+// TODO
```

Comment:
Removing this isn't really part of the changes needed here. I can help show you how to add particular things in Git, and we can do a new PR (with you as author), or I can rebase the changes (and I'd have to re-commit, making me the author..) Your choice :)